### PR TITLE
add openmind readme

### PR DESCRIPTION
**Pull request by baymax591**
Add openmind readme README_npu.

### CODE REVIEW

File: README_npu.md [hunk 1]
```diff
@@ -322,6 +322,96 @@ Execute the inference script:
...
+seed: 1234
+```
+
+Run the following command to start fine-tuning:
+
+```
+openmind-cli train examples/internlm3/train_sft_full_internlm3.yaml
+```
+
+# Accuracy and memory
```
Comment (A1waysBeenHere) [inline README_npu.md]: 是否改为Accuracy and Memory Consumption比较恰当

File: README_npu_zh-CN.md [hunk 2]
```diff
@@ -319,6 +319,95 @@ print(response)
...
+seed: 1234
+```
+
+通过以下命令行启动微调：
+
+```
+openmind-cli train examples/internlm3/train_sft_full_internlm3.yaml
+```
+
+### 精度显存
```
Comment (A1waysBeenHere) [inline README_npu_zh-CN.md]: 精度和显存开销？

File: README_npu.md [hunk 3]
```diff
@@ -322,6 +322,96 @@ Execute the inference script:
 python inference_internlm3_instruct_8b.py
 ```
 
+## openMind Library
+
+openMind Library provides an encapsulation of the Transformers Trainer class. You can use the openMind Trainer class in the same way as using the Transformers Trainer class, with no need to compile training loop steps before you start model training. The Trainer API supports various training options and functions, including logging, gradient accumulation, and model upload.
```
Comment (A1waysBeenHere) [inline README_npu.md]: ...., without needing to compile the training loop steps before starting model training.

File: README_npu_zh-CN.md [hunk 4]
```diff
@@ -319,6 +319,61 @@ print(response)
 python inference_internlm3_instruct_8b.py
 ```
 
+## openMind Library
+
+openMind Library基于transformers库，集成了PyTorch框架下主流第三方工具的功能，提供了一键式的模型微调解决方案，涵盖了从数据处理、多站点权重加载，到低参数微调（LoRA）、量化适配（QLoRA），以及微调和训练跟踪的全流程功能。同时，openMind Library还提供了昇腾NPU亲和算子优化等加速手段，显著提升模型训练效率。更多微调特性请参考[魔乐社区-模型微调-PyTorch模型微调](https://modelers.cn/docs/zh/openmind-library/1.0.0/basic_tutorial/finetune/finetune_pt.html)
```
Comment (baymax591) [inline README_npu_zh-CN.md:324]: openMind Library是一个开源的大模型套件，原生支持在昇腾NPU上进行微调、推理、评估和部署。openMind Library提供高易用性的接口和使用方式，充分发挥昇腾NPU的性能，快速支持、增强业界前沿模型。

File: README_npu_zh-CN.md [hunk 5]
```diff
@@ -319,6 +319,61 @@ print(response)
...
+```
+
+### 微调
+
+通过以下命令行启动微调：
+
+```
+openmind-cli train examples/internlm3/train_sft_full_internlm3.yaml
+```
+
```
Comment (baymax591) [inline README_npu_zh-CN.md:341]: openMind Library提供了昇腾NPU上的一键式模型微调方案，涵盖了数据处理、多站点权重加载，低参微调（LoRA）、量化适配（QLoRA）等能力。同时，openMind Library支持昇腾NPU融合算子优化，提升模型训练性能。更多特性请参考openMind微调文档。
